components/core/src/VariableDictionaryWriter.hpp

```diff
@@ -23,21 +23,12 @@ class VariableDictionaryWriter : public DictionaryWriter<variable_dictionary_id_
         }
     };
 
-    // Methods
     /**
-     * Opens dictionary, loads entries, and then sets it up for writing
-     * @param dictionary_path
-     * @param segment_index_path
-     * @param max_id
-     */
-    void open_and_preload (const std::string& dictionary_path, const std::string& segment_index_path, variable_dictionary_id_t max_id);
-
-    /**
-     * Adds an entry to the dictionary if it doesn't exist, or increases its occurrence count if it does
+     * Adds the given vartype string to the dictionary if it doesn't exist.
```

Comment:
How about `... given variable to ...`? The "type" in "logtype" refers to the type of log, whereas I think what you mean by "vartype" is that this string is a variable. Since the two usages of "type" are different, it might get confusing for the reader.

Reply:
I see, wasn't fully aware of this nuance before.